src/services/near-nep21-util.js

```diff
@@ -121,6 +121,23 @@ export async function gasCheck() {
   return false;
 }
 
+// Use decimal attribute format and return token amount.
+// Till 5 decimals
+export function convertToDecimals(str, decimals) {
+  let len = Number(decimals);
+  if(len == 0) {
+    return str;
+  }
+
+  let result = (str + "").padStart(len + 1, "0");
+  result = result.slice(0, -len) + "." + result.slice(-len);
+
+  if(decimals > 5) {
+    return result.slice(0, -(len - 5));
+  }
+  return result;
+}
```

Comment:
I don't think this works correctly. What if `str` already contains a dot?
Let's add test cases:
* convertToDecimals("1", 6) -> 0.000001
* convertToDecimals("11", 6) -> 0.000011
* convertToDecimals("1234567", 6) -> 1.234567
* convertToDecimals("1000000", 6) -> 1.0  // 1e6
* convertToDecimals("12000000", 6) -> 12.0 // 12e6
* convertToDecimals("12300000", 6) -> 12.3 // 12.3e6

Reply:
If str already has dot, then we won't use this function, we are using this function to convert values that come from contract.
and contract cannot return str with dot.
Addint test cases

Reply:
currently this function returns number with only 5 decimals.
for ex:  5.12344987395329 -> 5.12344
This is done for better readability in UI.

Reply:
Let's use ellipsis, and show the rest of the number when hovering on ellipsis.

Reply:
added using `ReactToolTip`